caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -2,6 +2,7 @@ import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { Panel } from 'react-bootstrap';
 import visMap from '../../../visualizations/main';
+require('../../../visualizations/directed_force.css');
```

Comment:
Turned out this line does not need to be used at all when className is specified :)
Deleted it!